### DISCUSSION TITLE
subtitleedit.exe - Portable Edition - Version 4.0.1 flagged by Microsoft Defender as Trojan

### DISCUSSION THREAD

**Discussion by ElPezP** [SubtitleEdit/subtitleedit#7415] [Q&A] [answered]
Hi, subtitleedit.exe - Version 4.0.1 just got quarantined by Microsoft Defender as Trojan:Win32/Bearfoos.A!ml 

And I tried to install the windows version but is also flagged by VirusTotal 😥

![image](https://github.com/SubtitleEdit/subtitleedit/assets/145389606/453885d8-6feb-4807-acbc-c9dcc9fe2100)

**Comment by Purfview** [comment, marked as answer]
https://www.microsoft.com/en-us/wdsi/filesubmission/

**Comment by niksedk** [comment]
It's unfortunately very common that new executable files are flagged. SE is flagged at every release. Normally only by 1-3 providers. 
(I stopped using an exe packer due to more than 20 warnings on virus total)

To help, please report SE as a false positive to your anti virus provider, thanks.
You should hear from them in about a week.

Do use Virus Total and common sense:
Portable: https://www.virustotal.com/gui/file/3729606b10a6cc47bc72c6ff1570931eaf1072974f979137ea6239ed6edada48
Installer: https://www.virustotal.com/gui/file/124a9dd46e041ed5fc05dc6341cf9ecc25ed5c7fa0b2512e42c20ff53240f005

**Comment by niksedk** [comment]
No flags on Virus total now :)
(check the above links)
